### DISCUSSION TITLE
A token grabber Phish on Github

### DISCUSSION THREAD

**Discussion by eericjacobson** [community/community#31959] [Other Feature Feedback, Questions, & Ideas]
So, i got dmed through discord. The dm went something like this.

"Do you want to play a game with me i am bored and there is this cool game you should download."

The account that contacted me is a hacked account of someone i dont know.

they then sent this github link and told me to download the game of there

quietforest2022.github.io/

i downloaded the game and put it through virus total and it came back as a Discord Token Grabber

This link could be harmful to many people and their discord accounts, so i propose that it is taken down immediately

**Comment by airtower-luna** [comment]
In that case it's best if you [report the user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam) for spreading malware.
